metadata/algolia/meta/system-objecttype-extensions.xml

```diff
@@ -175,6 +175,15 @@ Setting this preference replaces the first two segments, the final index name be
                 </value-definitions>
             </attribute-definition>
 
+            <attribute-definition attribute-id="Algolia_IndexOutOfStock">
```

Comment:
This preference has not been added to the Algolia group so it's not visible in the Custom Preferences of the site.

(side question: Do you know where I can see its value in that case?)

Reply:
answer for the side question: No, it is not possible via standard UI, only possible via our custom UI